frontend/src/Talk/Types.purs

```diff
@@ -16,19 +16,19 @@ data SelectedLang
   | OneLang String
   | TwoLang String String
 
-data Query a
-  = Init a
-  | OnClickLang String a
-  | OnClickPlay a
-  | HandleAudioProgress Web.Event a
-  | HandleAudioPlay Web.Event a
-  | HandleAudioPause Web.Event a
-  | HandleAudioError a
-  | OnToggleAudioControls a
-  | OnToggleAudioPlay a
-  | OnStopAudioPlay a
-  | OnAudioBackward a
-  | OnAudioForward a
+data Query 
```

Comment:
Can you please rename this to `data Action`? And add a `type Query = Const Void` line above.

Action is used internally, Query is exposed to parent component.